<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Клиенты</string>
     </attribute>
     <widget class="QTableView" name="tvClients">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddClient">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEditClient">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelClient">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRegisterVisit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Отметить посещение</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddSubscription">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Добавить абонемент</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Тренеры</string>
     </attribute>
     <widget class="QTableView" name="tvTrainers">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddTrainer">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelTrainer">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Абонементы</string>
     </attribute>
     <widget class="QTableView" name="tvSubTypes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddSubType">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelSubType">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>0</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Расписание</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarSchedule">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>312</width>
        <height>190</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddTraining">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Добавить тренировку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelTraining">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Отменить занятие</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvSchedule">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>281</width>
        <height>471</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Статистика/Финансы</string>
     </attribute>
     <widget class="QDateEdit" name="deFrom">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="deTo">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcStats">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>С какого числа</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>По какое число</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tvStatsDetails">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>221</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Финансовые итоги</string>
      </property>
      <widget class="QLabel" name="lblTotalRevenue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Выручка: [0 руб]</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblSoldCount">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Продано: [0 шт]</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
